AndroidManifest.xml

```diff
@@ -12,6 +12,8 @@
     <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
 
+    <uses-permission android:name="com.android.launcher.permission.INSTALL_SHORTCUT" />
```

Comment:
Found it:
https://developer.android.com/reference/android/Manifest.permission.html#INSTALL_SHORTCUT
INSTALL_SHORTCUT has protection level normal. When I install it is listed as a permission but it should not lead to a prompt for this permission.